apps/web/modules/videos/views/videos-single-view.tsx

```diff
@@ -351,34 +372,77 @@ export function LogInOverlay(props: LogInOverlayProps) {
     [error]
   );
 
+  const handleEmailChange = useCallback(
+    (e: React.ChangeEvent<HTMLInputElement>) => {
+      setUserEmail(e.target.value);
+      if (error) {
+        setError(null);
+      }
+    },
+    [error]
+  );
+
   return (
     <Dialog open={isOpen} onOpenChange={setIsOpen}>
       <DialogContent
         title={t("join_video_call")}
         description={t("choose_how_you_d_like_to_appear_on_the_call")}
-        className="bg-white text-black dark:bg-black dark:text-white sm:max-w-[480px]">
-        <div className="pb-4">
+        className="bg-black/80 p-6 sm:max-w-lg">
+        <div className="mt-2 pb-4">
           <div className="space-y-4">
             <div>
               <div className="font-semibold">{t("join_as_guest")}</div>
               <p className="text-subtle text-sm">{t("ideal_for_one_time_calls")}</p>
             </div>
 
-            <div className="flex gap-4">
-              <Input
-                type="text"
-                placeholder={t("your_name")}
-                className="w-full flex-1"
-                value={userName}
-                onChange={handleUserNameChange}
-                onKeyDown={handleKeyDown}
-                disabled={isLoading}
-                autoFocus
-              />
-              <Button color="secondary" onClick={handleContinueAsGuest} loading={isLoading}>
-                {t("continue")}
-              </Button>
-            </div>
+            {requireEmailForGuests ? (
```

Comment:
display email + name  when `requireEmailForGuests` is true and only name when `requireEmailForGuests` is false